src/renderer/angular/directives/commons/sk-tx-history.directive.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+function SKTxHistoryDirective($rootScope, $interval, RPCService, CommonService) {
+    'ngInject';
+
+    return {
+        restrict: 'E',
+        scope: {
+            tokenSymbol: '@'
+        },
+        link: (scope, element) => {
+            let publicKey = '0x' + $rootScope.wallet.getPublicKey();
+
+            let syncByWallet = () => {
+                RPCService.makeCall('syncTxHistoryByWallet', { walletId: $rootScope.wallet.id, publicKey });
+            };
+            syncByWallet();
+
+            scope.txList = [];
+            scope.tokenSymbol = scope.tokenSymbol ? scope.tokenSymbol.toUpperCase() : null;
+
+            let getTxStatusText = (tx) => {
+                let status = tx.txReceiptStatus;
+                let isSend = tx.from == publicKey;
+
+                if (status == 0) {
+                    return isSend ? 'Faild To Send' : 'Faild To Receive';
```

Comment:
Will users see this? If so, it would be nice to have translation engine up and running on these things.

Reply:
The entire UI needs localisation and internationalisation support added, but that's way out of scope for this PR.

Reply:
What does ``Sel 25`` stand for?